modules/0x01-OSINT+Footprinting/0x01-PassiveReconnaissance/getconinfo.py

```diff
@@ -9,69 +9,70 @@
 #This module requires TIDoS Framework
 #https://github.com/theInfectedDrake/TIDoS-Framework
 
+from __future__ import print_function
 import requests
 import time
 from colors import *
 
 def getconinfo(domain):
 
-	print R+'\n    ======================================='
-	print R+'     D O M A I N   C O N T A C T   I N F O'
-	print R+'    =======================================\n'
+	print(R+'\n    =======================================')
+	print(R+'     D O M A I N   C O N T A C T   I N F O')
+	print(R+'    =======================================\n')
 	time.sleep(0.6)
-	print GR+' [*] Importing API Key...'
+	print(GR+' [*] Importing API Key...')
 	try:
 		from files.API_KEYS import FULLCONTACT_API_KEY
-	except IOError, ImportError:
-		print R+' [-] Error while importing key...'
+	except IOError as ImportError:
```

Comment:
Something seriously went wrong here. :(

Reply:
Good catch!  The ambitiousness of the old style syntax is is why there was the move to new style.